apps/src/templates/manageStudents/AddMultipleStudents.jsx

```diff
@@ -0,0 +1,83 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {addMultipleAddRows} from './manageStudentsRedux';
+import Button from '../Button';
+import i18n from "@cdo/locale";
+import BaseDialog from '../BaseDialog';
+import DialogFooter from "../teacherDashboard/DialogFooter";
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  }
+};
+
+class AddMultipleStudents extends Component {
+  static propTypes = {
+    // Provided by redux
+    addMultipleStudents: PropTypes.func.isRequired,
+  };
+
+  state = {
+    openDialog: false,
+  };
+
+  openDialog = () => {
+    this.setState({openDialog: true});
+  };
+
+  closeDialog = () => {
+    this.setState({openDialog: false});
+  };
+
+  add = () => {
+    const value = this.refs.studentsTextBox.value;
+    this.props.addMultipleStudents(value.split("\n"));
+    this.closeDialog();
+  };
+
+  render() {
+    return (
+      <div>
+        <Button
+          onClick={this.openDialog}
+          color={Button.ButtonColor.gray}
+          text={i18n.addStudentsMultiple()}
+        />
+        <BaseDialog
+          useUpdatedStyles
+          uncloseable
```

Comment:
This dialog has a "Cancel" button.  Is there a good reason to make it uncloseable?  This is worth diverging from spec to provide user-friendly behavior, unless there's a specific reason for it.

Reply:
Nope - I copied this from somewhere and forgot to remove it.